runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -130,6 +131,10 @@ public XWalkContent(Context context, AttributeSet attrs, XWalkViewInternal xwVie
         XWalkPreferencesInternal.load(this);
     }
 
+    public View getContentView() {
```

Comment:
It's strange that getting a `View` object but the function name is `getContentView`. In this case, I think it's better to call `mContent.setOnTouchListener()`, and move the real implementation into `XWalkContent`